index.html

```diff
@@ -166,10 +152,9 @@ <h1>Welcome to Malcolm’s CSD120 Website!</h1>
               <td>40</td>
             </tr>
           </tfoot>
-          <caption>Table 1. Current Courses</caption>
```

Comment:
Caption must be the first child of a table element.  The caption-side property is how you get it to appear below